From be6d841a3bf5d5146603a9a2f5838dbcf6850287 Mon Sep 17 00:00:00 2001
From: "Tong.Chen" <tong.chen@windriver.com>
Date: Fri, 12 Oct 2018 15:58:11 +0800
Subject: [PATCH 02/11] Revert "driver: clk: socfpga: remove unused variable"

This reverts commit 97e710ef0545c19d3c10bd81a61bdca9fe543b81.

Get patches related with stratix10 clock manager code from sdk kernel
v4.18 to update clock manager drvier, so revert patches of old clock
manager drvier.

Signed-off-by: Tong.Chen <tong.chen@windriver.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/socfpga/clk-pll-s10.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/clk/socfpga/clk-pll-s10.c b/drivers/clk/socfpga/clk-pll-s10.c
index 231c798..52a7637 100644
--- a/drivers/clk/socfpga/clk-pll-s10.c
+++ b/drivers/clk/socfpga/clk-pll-s10.c
@@ -93,7 +93,7 @@ static int clk_pll_prepare(struct clk_hw *hwclk)
 static struct clk * __init __socfpga_pll_init(struct device_node *node,
 					      const struct clk_ops *ops)
 {
-	u32 reg;
+	u32 reg, reg2;
 	struct clk *clk;
 	struct socfpga_pll *pll_clk;
 	const char *clk_name = node->name;
-- 
1.7.9.5

